MobileBuy/buy/build.gradle

```
@@ -128,12 +128,6 @@ dependencies {
     androidTestCompile 'com.google.dexmaker:dexmaker:1.1'
     androidTestCompile 'com.google.dexmaker:dexmaker-mockito:1.1'
 
-
-    testCompile 'org.mockito:mockito-core:1.9.5'
```

I think the testUsingMockito should fail if you are missing this import